tests/drivers/timer/nrf_rtc_timer/src/main.c

```diff
@@ -38,8 +38,8 @@ static void init_zli_timer0(void)
 
 	IRQ_DIRECT_CONNECT(TIMER0_IRQn, 0,
 			   timer0_isr_wrapper,
-			   COND_CODE_1(CONFIG_ZERO_LATENCY_IRQS,
-				   (IRQ_ZERO_LATENCY), (0)));
+			   IS_ENABLED(CONFIG_ZERO_LATENCY_IRQS) ?
```

Comment:
this may be kept the same, as `#define _EXC_SVC_PRIO COND_CODE_1(CONFIG_ZERO_LATENCY_IRQS, (1), (0))` uses `COND_CODE_1` now.

Reply:
Frankly speaking, i prefer `IS_ENABLED` since it is using compiler. Previously, `COND_CODE_1` had to be used because `IRQ_ZERO_LATENCY` did not exist when ZLI was disabled. I see it as beneficial that `IS_ENABLED` can be used in cases like this.

Reply:
fair enough